crates/volta-core/src/tool/npm/mod.rs

```diff
@@ -46,3 +89,87 @@ impl Display for Npm {
         f.write_str(&tool_version("npm", &self.version))
     }
 }
+
+/// The Tool implementation for setting npm to the version bundled with Node
+pub struct BundledNpm;
+
+impl Tool for BundledNpm {
+    fn fetch(self: Box<Self>, _session: &mut Session) -> Fallible<()> {
+        info!("Bundled npm is included with Node, use `volta fetch node` to fetch Node");
+        Ok(())
+    }
+
+    fn install(self: Box<Self>, session: &mut Session) -> Fallible<()> {
+        let toolchain = session.toolchain_mut()?;
+
+        toolchain.set_active_npm(None)?;
+
+        let bundled_version = toolchain.platform().and_then(|platform| {
+            // If we can't load the default Npm version, treat that as no npm being available
+            let version = load_default_npm_version(&platform.node).ok()?;
+            Some(version.to_string())
+        });
+
+        match bundled_version {
+            Some(version) => {
+                info!(
+                    "{} set bundled npm (currently {}) as default",
+                    success_prefix(),
+                    version
+                );
+            }
+            None => info!("{} set bundled npm as default", success_prefix()),
```

Comment:
Let's show an error here: maybe prompt to retry in case there was a transient issue, but at a minimum let them know that there *was* an issue (maybe corrupt installation or something?).

Reply:
Updated to show an error that prompts the user to install / pin a Node version. The `node@bundled` operation completed successfully at this point, so retrying it wouldn't change anything. But they won't be able to run `npm` until they have a valid Node version, so that seems like the most useful path forward.